internal/namespaces/rdb/v1/custom_backup.go

```diff
@@ -119,6 +119,68 @@ func backupRestoreBuilder(c *core.Command) *core.Command {
 	return c
 }
 
+func backupListBuilder(c *core.Command) *core.Command {
+	type customBackup struct {
+		ID         string                   `json:"ID"`
+		Name       string                   `json:"name"`
+		InstanceID string                   `json:"instance_ID"`
+		Exported   bool                     `json:"exported"`
+		Status     rdb.DatabaseBackupStatus `json:"status"`
+	}
+
+	c.View = &core.View{
+		Fields: []*core.ViewField{
+			{
+				Label:     "ID",
+				FieldName: "ID",
+			},
+			{
+				Label:     "Name",
+				FieldName: "Name",
+			},
+			{
+				Label:     "Status",
+				FieldName: "Status",
+			},
+			{
+				Label:     "Instance ID",
+				FieldName: "InstanceID",
+			},
+			{
+				Label:     "Exported",
+				FieldName: "Exported",
+			},
+		},
+	}
+
+	c.AddInterceptors(func(ctx context.Context, argsI interface{}, runner core.CommandRunner) (i interface{}, err error) {
+		listBackupResp, err := runner(ctx, argsI)
+		if err != nil {
+			return listBackupResp, err
+		}
+		backupList := listBackupResp.([]*rdb.DatabaseBackup)
+		var res []customBackup
+		for _, backup := range backupList {
+			expired := false
+			if backup.DownloadURLExpiresAt == nil {
+				expired = true
+			} else {
+				expired = time.Now().After(*backup.DownloadURLExpiresAt)
+			}
+			res = append(res, customBackup{
+				ID:         backup.ID,
+				Name:       backup.Name,
+				Status:     backup.Status,
+				InstanceID: backup.InstanceID,
+				Exported:   !expired,
```

Comment:
Could you manipulate an exported variable that will get modified across the function instead of an expired variable that sounds similar but is different?

Reply:
You could also export this functionality in an isExported function